ambry-api/src/main/java/com/github/ambry/store/StoreGetOptions.java

```diff
@@ -25,8 +25,12 @@ public enum StoreGetOptions {
   Store_Include_Expired,
   /**
    * This option indicates that the store needs to return the message even if it has been
-   * marked for deletion as long as the message has not been physically deleted from the
-   * store.
+   * marked for deletion as long as the message is still in delete retention window
    */
-  Store_Include_Deleted
+  Store_Include_Deleted,
+  /**
+   * This option indicates that the store needs to return the message as long as the message
```

Comment:
QQ. Do we need this new option? Would it be easier if we tell customers that `Store_Include_Deleted` only works if they issue GET within the delete retention window?

Reply:
The reason to add this option is to make sure that Include_All get option would continue to work for compaction_ready blobs.